opt/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -515,6 +515,12 @@ def voteAnnotation(self, request, context):
         success, message = loom.annotation_vote(request, self.config["dataHashSecret"])
         return s_pb2.voteAnnotationReply(success=success, message=message)
 
+    def getClusterOverlaps(self, request, context):
+        """Get overlapping clusters of the requested cellIDs and return formatted protobuf object"""
```

Comment:
[PEP-257](https://www.python.org/dev/peps/pep-0257/) says we should end doc strings with a full-stop. Should actually be caught by `pylint`...